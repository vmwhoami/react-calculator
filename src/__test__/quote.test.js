import React from 'react';
import renderer from 'react-test-renderer';


it('renders quote page correctly', () => {
  const tree = renderer
    .create(
      <main className="maincont">
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit.
          Cum odit doloremque laboriosam similique inventore amet,
          magni accusamus veniam saepe asperiores eveniet
          voluptatem dolorum quaerat animi neque, molestias
          consequatur aut quis?
</p>
      </main>


    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});


