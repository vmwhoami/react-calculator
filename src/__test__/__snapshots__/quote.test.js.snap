// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders quote page correctly 1`] = `
<main
  className="maincont"
>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Cum odit doloremque laboriosam similique inventore amet, magni accusamus veniam saepe asperiores eveniet voluptatem dolorum quaerat animi neque, molestias consequatur aut quis?
  </p>
</main>
`;
